From 509beab8c9ecca1548cb3f4e01dedfcae1cd68fc Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Fri, 9 Nov 2018 16:39:41 +0800
Subject: [PATCH 346/706] arm64: dts: ls1028: Fix errata TKT362913 for ARM Mali
 DP500

On LS1028A ArchDef rev 1.0, allow the LCD Controller Controller
Clock PCLK = ACLK.

Signed-off-by: Wen He <wen.he_1@nxp.com>
(cherry picked from commit b174c563a1aa337f14599562f205ac08ea4e1b13)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 1d063fd9f39c..f321153cef3a 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -136,7 +136,7 @@
 	pclk: pclk {
 		compatible = "fixed-clock";
 		#clock-cells = <0>;
-		clock-frequency = <325000000>;
+		clock-frequency = <650000000>;
 		clock-output-names= "pclk";
 	};
 
-- 
2.17.1

